packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.service.js

```diff
@@ -20,10 +25,28 @@ export default class {
     this.OvhApiSupport = OvhApiSupport;
   }
 
-  getServices(route) {
+  static addSubTypes(serviceTypes) {
+    forEach(SUB_SERVICE_TYPES, (subTypes, serviceName) => {
+      const service = head(
+        remove(serviceTypes, (serviceType) => serviceType.name === serviceName),
+      );
+      if (service) {
+        forEach(subTypes, (subType) => {
+          serviceTypes.push({
```

Comment:
Is `filter` + `map`  possible instead ?

Reply:
Here `serviceTypes` is an existing array. Hence we would not be able to use `map`. We could have used it if we are creating a new array :)